gender_novels/analysis/analysis.py

```diff
@@ -158,7 +162,80 @@ def display_gender_freq(d, title):
     plt.savefig(filepng, bbox_inches='tight')
     plt.savefig(filepdf, bbox_inches='tight')
 
-if __name__ == '__main__':
-    test_function()
+def dunn_individual_word(m_corpus,f_corpus,word):
```

Comment:
dunn_individual should take as params:
`word_count_corpus_1 : int, word_count_corpus_2 : int, total_words_corpus_1 : int, total_words_corpus_2: int`
i.e. the only thing you're passing in are four integers. 
Right now, it's loading the corpus for every single word.